server/src/main/java/org/opensearch/extensions/rest/RestExecuteOnExtensionRequest.java

```diff
@@ -57,21 +63,25 @@ public String getUri() {
         return uri;
     }
 
+    public PrincipalIdentifierToken getToken() {
```

Comment:
While the token's type is very illustrative of what it is, `token` doesn't convey much meaning, how about getRequestIssuerIdentity()?

Reply:
I will leave it to getToken here unless we want to rename the field to `requestIssuerIdentity`, but i will rename `generateToken` to `generateRequestIssuerIdentity`. Thoughts?